core/src/modules/ops/recordProperty.ts

```diff
@@ -10,6 +9,7 @@ import { Source } from "../../models/Source";
 import { AggregationMethod, PluginConnection } from "../../classes/plugin";
 import { log } from "actionhero";
 import { PropertiesCache } from "../../modules/caches/propertiesCache";
+import { plugin } from "../plugin";
```

Comment:
The import was moved because I removed the phone number formatting, and then manually brought it back. It's in use [here](https://github.com/grouparoo/grouparoo/pull/2971/files#diff-f26d235951e3516c278cdfcefb9e8cbc54e164a23aec8563430624cdf5d14546R295)